test/integration/manifests/noop-deployment-windows.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     spec:
       containers:
       - name: noop
-        image: mcr.microsoft.com/windows/nanoserver:ltsc2022
+        image: mcr.microsoft.com/oss/kubernetes/pause:3.6

```

Comment:
instead of just changing this image, should evaluate if this _deployment_ needs to be OS-specific or if there should only be one using (this) multiplat image

Reply:
Will test on a separate branch and communicate with @vipul-21 to see what we should do going forward.